pages/common/keepass2.md

```diff
@@ -0,0 +1,15 @@
+# keepass2
+
+> A light-weight password manager.
+
+- Start KeePass 2, opening the most recently-opened password database:
+
+`keepass2`
+
+- Start KeePass 2, opening a specific password database:
+
+`keepass2 "{{/path/to/database.kbdx}}"`
+
+- Use a specific key file to open a password database:
+
+`keepass2 "{{/path/to/database.kbdx}}" -keyfile:{{/path/to/key/file.key}}`
```

Comment:
Shouldn't there be quotes around the key file path too ?

And how is the key file generated for a particular db ?